pkg/sentry/platform/kvm/machine_arm64_unsafe.go

```diff
@@ -47,6 +48,16 @@ func (m *machine) initArchState() error {
 		uintptr(unsafe.Pointer(&vcpuInit))); errno != 0 {
 		panic(fmt.Sprintf("error setting KVM_ARM_PREFERRED_TARGET failed: %v", errno))
 	}
+
+	// initialize all vCPUs
+	m.initialvCPUs = make(map[int]*vCPU)
+	for int(m.nextID) < m.maxVCPUs-1 {
+		m.mu.Lock()
```

Comment:
I am not sure that we need to take this mutex here. And even if we need, we can take it for the whole loop.